### ISSUE TITLE
Plask profling

### ISSUE THREAD

**Issue by vorg** [deanm/plask#39]
I've been following [this guide](https://github.com/thlorenz/v8-perf/issues/4) on how to make a flame chart from node v8.log profile and they mention

> With a version of node that has a **--log-timer-events** as part of **v8-options**, run the app in profiling mode and load the resulting v8.log into Chrome at chrome://tracing.

Is `--log-timer-events` a compilation option? My graphs are empty so not sure if these log timer evens are properly logged.

Any other ways to profile a Plask app?

**Comment by deanm** [comment]
Yeah, I've profiled plask many times, in general --prof works and writes out v8 log files, but there will be more than one log file, because the log file is per context.  You need a tick processor that matches the right version of v8, which is the only tricky bit, but I've done it a few times before and gotten good profiles.
